server/common/diffexpdu/postingslist.py

```diff
@@ -340,7 +340,9 @@ def _un_byteshuffle(buf: Union[bytes, bytearray, memoryview], dtype) -> np.ndarr
 
 
 def _delta(arr: np.ndarray) -> np.ndarray:
-    return np.diff(arr, prepend=arr.dtype.type(0))
```

Comment:
before this change tests such as `def test_roundtrip()` would fail for precision 0.999